### PR TITLE
feat(recording): upd to 2023

### PR DESCRIPTION
**Pull request by tim08094495757**
## Types of changes

* **New feature**

## Description

update i18n, font-size, padding, structure
## Steps to Test This Pull Request

1. go to /speaking/recording
2. test languages (zh-hant, en-us)
3. test screen width (@min-width: 768px, 1024px)

## Related Issue

#341

### CODE REVIEW

File: i18n/speaking/recording.i18n.js [hunk 1]
```diff
@@ -3,18 +3,21 @@ import { genI18nMessages } from '~/utils/i18n.utils'
...
+        intro: [
             'Since one of the PyCon motivations is to help Python education and advocate the use of Python. ' +
                 'PyCon Taiwan Committee will record, edit, and release the audio and video of the speaker’s presentation to online video sites ' +
                 'such as YouTube and {pyvideo} by default. ',
-            'Thanks to our sponsors, we could make PyCon APAC one of the greatest events. ' +
-                "We will embed sponsors' logos in the presentation videos that play during and after the conference. " +
-                'If you don’t want your talk being uploaded after the conference, make sure you choose “No” to the “Recording Release” ' +
-                'option during the proposal submission.',
+        ],
+        policies: [
```
Comment (tim08094495757) [inline i18n/speaking/recording.i18n.js:11]: imitate: 
https://github.com/pycontw/pycontw-frontend/blob/main/i18n/speaking/tutorial.i18n.js

**Comment by tim08094495757** [comment]
I try to test codespace xd. How can I remove this [29ea9fe](https://github.com/pycontw/pycontw-frontend/pull/358/commits/29ea9fec917e859b54072ace5a42f1776d70f26d) ?

**Comment by mattwang44** [comment]
@tim08094495757 you can consider using `git rebase` to drop the commit

**Comment by tim08094495757** [comment]
> @tim08094495757 you can consider using `git rebase` to drop the commit

THX.

File: pages/speaking/recording.vue [hunk 2]
```diff
@@ -55,4 +78,17 @@ export default {
...
 
-<style scoped></style>
+<style lang="postcss" scpoed>
+.recording__intro {
+    @apply text-[18px];
+    @media (min-width: 1024px) {
+        @apply text-base;
+    }
+}
+.recording__two-col-wrapper {
```
Comment (tim08094495757) [inline pages/speaking/recording.vue:88]: After creating a new PR, I think I will rewrite this part later. Like tutorial__policy in: https://github.com/pycontw/pycontw-frontend/pull/359/commits/fcb749564c66b6a7afb7904c7d8b2f47f2b66a85

**Review by josix — APPROVED** [review]
LGTM!

File: pages/speaking/recording.vue (hunk 2 above)
Comment (tim08094495757) [reply pages/speaking/recording.vue]: fixed: [ab21f13](https://github.com/pycontw/pycontw-frontend/pull/358/commits/ab21f1349df3ff0e591d7dbbe6a151cab53fff7e)

**Pull request merged**